### PR TITLE
Fix Bug 1217214, update first blurb to uppercase Web

### PR DESCRIPTION
**Pull request by schalkneethling**
@flodolo r?

### CODE REVIEW

File: bedrock/firefox/templates/firefox/choose.html [hunk 1]
```diff
@@ -33,7 +33,13 @@ <h2 data-tracking="{{ _('Browse Freely.<br> Choose Firefox.') }}" data-choice="{
 <section class="page-content">
   <div class="blurb">
     <h2>{{ _('The most private browsing') }}</h2>
-    <p>{{ _('Take control of your web by using Private Browsing with Tracking Protection, only from Firefox.') }}</p>
+    <p>
+      {% if LANG == 'en-US' %}
+      {{ _('Take control of your Web by using Private Browsing with Tracking Protection, only from Firefox.') }}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/choose.html]: Nit: fix indentation

File: bedrock/firefox/templates/firefox/choose.html [hunk 2]
```diff
@@ -33,7 +33,13 @@ <h2 data-tracking="{{ _('Browse Freely.<br> Choose Firefox.') }}" data-choice="{
 <section class="page-content">
   <div class="blurb">
     <h2>{{ _('The most private browsing') }}</h2>
-    <p>{{ _('Take control of your web by using Private Browsing with Tracking Protection, only from Firefox.') }}</p>
+    <p>
+      {% if LANG == 'en-US' %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/choose.html]: Could show this string for any locale that starts with `en-` instead of just `en-US`?

Comment (flodolo) [inline bedrock/firefox/templates/firefox/choose.html]: We have en-GB, but I don't see anything that requires a change in this sentence, so OK for en-*

**Comment by alexgibson** [comment]
r+ care to squash?

**Pull request merged**